systers_portal/static/css/style.css

```diff
@@ -741,3 +741,14 @@ table.decoration-none  tr a {
         display: none;
     }
 }
+input[id=form1] {
+  width: 130px;
+  height: 40px;
+  -webkit-transition: width 0.4s ease-in-out;
+  transition: width 0.4s ease-in-out;
```

Comment:
Can we let the animation be? None of the other pages have animation and especially when the search is completed it shrinks back and I don't think that looks that good. @ritwickraj78

Reply:
Everything else LGTM

Reply:
@sakshi1499 Sure